lib/browser/RemoteScriptRunner.js

```diff
@@ -138,7 +139,11 @@ module.exports = function () {
 
     RemoteScriptRunner.prototype.nextCommand = function () {
         if (this._cmdQueue.length > 0) {
-            return this._cmdQueue.shift();
+            var command = this._cmdQueue.shift();
+            if(this._debug) {
+                console.log("RemoteScriptRunner received command: " + command);
```

Comment:
This code runs in the browser and might not work for all browsers, see: http://patik.com/blog/complete-cross-browser-console-log/.  This is okay for now because it doesn't get executed by default, but we will ant to fix this at somepoint